Off-Top-App/off-top-flutter
# OT-38|FE - Hook up recording API in Flutter.io

**What I did:** 
1. Set up the audio recorder
2. Refactor file names and folder structure
3. Refactor the main recorder page as well as renaming to recordingSession.dart

How to test: 
1. Go to off-top-flutter in your terminal and run `flutter packages get` to make sure you have all packages.
2. Once everything looks okay, run `flutter run` once you have the emulator up and running.
3. Once you click on the microphone icon, if you see the timer increasing, and a red/green bar on the bottom fluctuating based on what I think is decibel levels then it works.
<img width="1680" alt="Screen Shot 2020-04-15 at 1 10 41 AM" src="https://user-images.githubusercontent.com/26909882/79313943-f0a8dc00-7eb5-11ea-86b3-dbfd91f1d09c.png">
![Screen Shot 2020-04-15 at 1 10 41 AM (2)](https://user-images.githubusercontent.com/26909882/79313952-f30b3600-7eb5-11ea-95ec-5f71826f5ad3.png)
